cmake/Dependencies.cmake

```diff
@@ -93,6 +93,26 @@ if(USE_CUDA)
   endif()
 endif()
 
+# ---[ XPU
+if(USE_XPU)
+  set(CAFFE2_USE_XPU ${USE_XPU})
+  include(${CMAKE_CURRENT_LIST_DIR}/public/xpu.cmake)
+  if(NOT CAFFE2_USE_XPU)
+    message(WARNING "Not compiling with XPU. Could NOT find SYCL (${IntelSYCL_NOT_FOUND_MESSAGE})."
```

Comment:
Please rename `IntelSYCL_NOT_FOUND_MESSAGE}` to `SYCL_NOT_FOUND_MESSAGE`

Reply:
done.